### PR TITLE
Br72 test module loss function

### PR DESCRIPTION
**Pull request by zengxyu**

### CODE REVIEW

**Review by le1nux — CHANGES_REQUESTED** [review]
Left a few comments. I think we can merge this soon :)

File: pytest/loss_functions/test_loss_functions.py [hunk 1]
```diff
@@ -13,15 +14,19 @@ class TestLPLossFunctions:
 
     @pytest.fixture
     def inference_batch_result_train(self) -> InferenceResultBatch:
```
[the remark this answers is not in the document]
Comment (zengxyu) [reply pytest/loss_functions/test_loss_functions.py:16]: done

File: pytest/loss_functions/test_loss_functions.py [hunk 2]
```diff
@@ -13,15 +14,19 @@ class TestLPLossFunctions:
...
-        predictions = torch.ones(size=[TestLPLossFunctions.batch_size, TestLPLossFunctions.sample_size])*3
-        return InferenceResultBatch(targets={TestLPLossFunctions.target_key: targets}, predictions={TestLPLossFunctions.prediction_key: predictions}, tags=None)
+        targets = torch.ones(
+            size=[TestLPLossFunctions.batch_size, TestLPLossFunctions.sample_size]) * 2  # [batch_size, sample_size]
+        predictions = torch.ones(size=[TestLPLossFunctions.batch_size, TestLPLossFunctions.sample_size]) * 3
+        return InferenceResultBatch(targets={TestLPLossFunctions.target_key: targets},
+                                    predictions={TestLPLossFunctions.prediction_key: predictions}, tags=None)
 
     @pytest.fixture
     def inference_batch_result_test(self) -> InferenceResultBatch:
```
[the remark this answers is not in the document]
Comment (zengxyu) [reply pytest/loss_functions/test_loss_functions.py:24]: done

File: pytest/loss_functions/test_loss_functions.py [hunk 3]
```diff
@@ -126,3 +140,9 @@ def test_bce_with_logits_loss(self, bce_with_logits_inference_batch_result):
                                      prediction_subscription_key=TestLPLossFunctions.prediction_key)
         loss = loss_fun(bce_with_logits_inference_batch_result)
         assert True
```
[the remark this answers is not in the document]
Comment (zengxyu) [reply pytest/loss_functions/test_loss_functions.py:142]: done

File: pytest/loss_functions/test_loss_functions.py [hunk 4]
```diff
@@ -126,3 +140,9 @@ def test_bce_with_logits_loss(self, bce_with_logits_inference_batch_result):
...
+
+    def test_bce_loss(self, bce_loss_inference_batch_result):
+        loss_fun = BCELoss(target_subscription_key=TestLPLossFunctions.target_key,
+                           prediction_subscription_key=TestLPLossFunctions.prediction_key)
+        loss = loss_fun(bce_loss_inference_batch_result)
+        assert True
```
[the remark this answers is not in the document]
Comment (zengxyu) [reply pytest/loss_functions/test_loss_functions.py:148]: done

File: pytest/loss_functions/test_multi_loss.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+from ml_gym.loss_functions.loss_functions import LossWarmupMixin
+from ml_gym.loss_functions.multi_term_loss_functions import MultiLoss
+
+# todo waiting the implementaion of MultiLoss.warm_up() and MultiLoss.finish_warmup()
```
[the remark this answers is not in the document]
Comment (zengxyu) [reply pytest/loss_functions/test_multi_loss.py:4]: already deleted the files

**Comment by le1nux** [comment]
Awesome. Looks good to me! :-)
